### DISCUSSION TITLE
Fargebruk

### DISCUSSION THREAD

**Discussion by onordbo** [fremtind/jokul#2335] [Annet]
Som et ledd i [inventory prosjektet](https://github.com/fremtind/jokul/projects/6) vårt ønsker vi tilbakemelding og stemmer fra både design og teknologi for å kartlegge hvilke komponenter og behov vi burde prioritere. Nedenfor følger en kort hypotese/slutning fra Designsystem-teamet, i tillegg til en rekke kommentarer vi har merket oss gjennom arbeidet.

Se på skjermbildene fra inventory [tilgjengelig her for referanse](https://www.figma.com/file/66kOJ5nCHjgtkfLW2sQWB8/Jøkul-Inventory-Innsiktsyntese?node-id=0%3A1). Legg gjerne til ønsker, behov, tanker, meninger og annet som kommenaterer i denne tråden, så vi kan sørge for at oppgavene blir så godt beriket med informasjon som mulig, dette gjør det enklere for alle involverte å ta tak i og løse de utfordringene vi møter sammen.

***

### TL;DR 👉 Hovedsakelig god implementering av brand farger og light/dark-mode, dog noen “avvik” her og der.

- Gjengkjennelig typografi på tvers av løsningene som bruker Jøkul
- Noe Jøkul 1.0 og FFE
- Større bruk av ikoner, farger på internsystemer for å legge til rette for “scanning” av mye innhold
- Variasjon i hva som er “hovedbakgrunnsfarge”
- BM: Fargeprofil er 1.0/gammel, checkbox/fieldgroup, select
- SeOpp: Ny fargeskala og ny typografiskala er satt/kartlagt, men som ikke er tatt i bruk enda
- Flyt (tabell): En farge som ikke var Jøkul ble brukt i en tabell, 50% snøhvit for synlighet/kontrast
- Dark-mode/light-mode er enten avskrudd eller hardkodet inn i noen løsninger. (hhv. SAM, BSF, Flyt)
- Dark mode er ikke like populært blant superbrukere av internesystemer (vanskelig å lese/bruke over tid/trist å se på hver dag)
- Motor: hover-state på kort i dark mode kan oppleves utydelig
- SeOpp: hvis man kommer fra SB1 eller DNB så er det litt ekstra co-branding på toppen i portalen. Burde dette strammes opp, i så fall hvordan?
- Flyt (behov/ønske/utfordring): scannbarhet blir litt hemmet i Flyt pga Jøkul sin fargepalett, dette ser vi blir en utfordring i tabeller. Begrenset fargepalett og ikonsett. Lite som skiller seg ut visuelt.
- PM: Skisser designes i jøkul 2.0 per i dag, men implementeringen er ikke fra jøkul, dog er den visuelt stylet likt.
- Motor følger et innrammingsprinsipp, men fortsatt blå chatteboble i kundestøtte?
- SeOpp: fortsatt blå/sb1 branding
- Smartforhandler: Har brukt rødfarge i deler av sidetittel, men skal bort

**Comment by wkillerud** [comment]
### Variasjon i hva som er hovedbakgrunnsfarge

Profilen sier "[d]et etterlatte inntrykket av våre løsninger skal være snøhvit med granitt som profilbærende farger", "Snøhvit bruker vi mest som bakgrunnsfarge", og "Varde bruker vi som bakgrunnsfarge eller på elementer **for å fremheve innhold**" (min utvehing). Mulig det har sklidd litt ut i noen av løsningene.

Trenger vi noen endringer i dokumentasjonen for å gjøre det tydeligere hva som skal være hovedbakgrunnsfargen?

**Comment by wkillerud** [comment]
### Hardkodet mørkt eller lyst tema

- Vi nevner ikke eksplisitt i fargedokumentasjonen at temafargen skal følge _brukerens_ systemvalg. Burde vi det?
- Når er det greit å overstyre brukerens valg?

**Comment by onordbo** [comment]
@Slaine2 har jobbet videre med dette i det siste, har du mulighet til å dele en oppdatering med oss her? Eventuelt lage en issue basert på arbeidet ditt så langt?

**Reply by onordbo** [reply]
Det ser ut som det pågår litt arbeid på dette allerede her: #2457

**Comment by Slaine2** [comment]
På VD-Forum har vi nå åpnet vinduet igjen og tenkt på den beste måten å bruke hjelpeteksten på med tanken om UU.

Vi har kommet frem til at vi virkelig ønsker å teste dette, for vi har x forskjellige meninger. Ida gjør en brukertest og gi oss tilbakemelding.
